src/VideoManager/VideoManager.cc

```diff
@@ -110,6 +110,7 @@ VideoManager::setToolbox(QGCToolbox *toolbox)
 #ifndef QGC_DISABLE_UVC
    // If we are using a UVC camera setup the device name
    _updateUVC();
+   emit isUvcChanged();
```

Comment:
That is happening on:
https://github.com/mavlink/qgroundcontrol/pull/10349/files#diff-1e8e0d1887083c7f6f0dd38e78042c25afb687cc60b955bf572f039ed9547fd0R493

Reply:
@patrickelectric 
> The video stream is not working here on both APK and also native build: 

1) it should work on native build - what OS do you have? 
2) it might not work on Android since a) I did not test it - sorry for that - **I will test it** b) I rebased to master and there gstreamer does not work as I mentioned in conversation with @fredowski - **I will check it and fix** prior to this PR so we have clean resolution before my tidy works in current PR

Thank you for rest of the feedback. I will fix it.

What about notices by @WTPENGUIN Shall I revert changes in `ts` files? I don't know what are translation mechanisms and maybe he's right and I shall not touch `ts` files - which would be strange though.

Reply:
fixed